Solo girl in the Dakshinkashi.txt

```diff
@@ -0,0 +1,19 @@
+
+I have a long list of places to visit and not all our famous touristy spots, I am a lover of infamous small towns and love to watch the daily hustle of the localities. My plans were often being cancelled or postponed due to unavailability at least one friend. That is when I decided. If you want to see a place. You get there! And that is how this this random Solo trip to Temple town of Kalasa happened!
+Kalasa popularly known as Dkashinakashi is a town in Chikkamagaluru district is surrounded by western ghats and lies on the banks on Bhadra river, the city's main attraction is the Shri Kalaseshwara Swami Temple built by Cholas. It is believed that saint Agasthya witnessned the wedding of Shiva and Parvathy devi from here. 
+The town itself is a quite with not many tourist around, specially in the monsoons. 7kms from here is the holy place of Horanadu, carpeted with green paddy fields and accompanied by a hilly backdrop stands the Adhishakthyathmaka Sri Annapoorneshwari Ammanavara Temple. It is said that one who visits this temple will never have shortage of food in their lives. 
+Having the annaprasadam at the temple and walking in the streets of Horanadu, the bus journey crossing the bhadra river while it drizzled all way along felt magical. 
+
+Kalasa has pancha teerthas that are pilgrimage route, of which I visited Vasistha Theertha, dedicated to the sage Vasistha. Having no vehicle of my own, I decided to walk to the teertha which is almost 2kms from the temple. The road is surrounded by forest and coffee estates. For a person who is not familiar with the western ghat's estates, this could a little overwhelming to walk alone. 
+After observing my many scared attempts of crossing a curvy bent road covered with tall trees and hills,  a kind localite offered to drop me till the bridge.
+
+The Hanging Bridge was one of the most beautiful things one could witness, the roaring bhadra river, picturesque back drop of Kudremukha mountain ranges kissed by the monsoon clouds and the misty rains felt like a scene straight out of a Yogaraj Bhat movie!
+When set on a solo adventure, the little interactions we get to make with local people gives you the best memories to cherish. If you pay attention to your surroundings there are so many people that you can strike a conversation with, the hotel manager who suggests you places to visit, the local tea shop couple who are interested about your journey, old bus conductor sharing life lessons or the coffee shop anna who willing helped me figure out bus back home. 
+
+All in all, If you have a long awaited trip to take, this time go solo and you will start seeing world from a whole new perspective and trust me you are going to love it.!
```

Comment:
tavelling locally can be explained in this blog like a gist. ex. travelling in a local bus to horandu from kalasa..... etc